src/sentry/monkey/pickle.py

```diff
@@ -127,4 +150,5 @@ def py3_compat_kombu_pickle_loads(s, load=__py3_compat_kombu_pickle_load):
         return original_kombu_pickle_loads(s, load)
 
     pickle.loads = py3_compat_pickle_loads
+    pickle.dumps = py3_compat_pickle_dumps
```

Comment:
Will this work with kombu as well?

Reply:
I've added `kombu_serializer.pickle_protocol = 2` to handle this